schema/deploy/tables/application_003.sql

```diff
@@ -0,0 +1,23 @@
+-- Deploy ggircs-portal:tables/application_003 to pg
+-- requires: tables/application_002
+
+begin;
+
+alter table ggircs_portal.application add column swrs_report_id integer;
+create unique index application_swrs_report_id_uindex on ggircs_portal.application(swrs_report_id);
```

Comment:
Are we sure that we want a unique constraint here? 
It makes sense that we would not see two applications with the same swrs_report_id, but I am wondering what situation could trigger an error here, and how our application would handle that error. We already have a constraint on the (facility_id, reporting_year) pair if I remember correctly

Reply:
We do have a uindex on the (facility_id, reporting_year) pair. I figured we'd probably want a unique constraint on the swrs_report_id since there should only ever be 1 per application. But I can take it out if you think it could be a problem somewhere?

Reply:
The `swrs_report_id` **should** be unique, but there is once instance of historical data (from 2011) where that is not the case. Given that this value is assigned by a third party service, it would probably be better to not create unique constraints in our DB based on it, so that we are more resilient to a potential bug in SWRS